app/mobile-content/exhibits/shared/exhibit.service.ts

```diff
@@ -0,0 +1,126 @@
+import { Injectable } from '@angular/core';
+import { Response } from '@angular/http';
```

Comment:
The indentation has been fixed the order of the imputs in this fine seems to be ok? or am i missing something